client/src/components/Benefits/benefits.ts

```diff
@@ -0,0 +1,18 @@
+export default [
```

Comment:
if you type this fella as `Benefit[]` you won't have to elsewhere

Reply:
Good 2 know! Will update